themes/saleor/js/app.js

```diff
@@ -1,3 +1,36 @@
-$(document).ready(function(){
-     $("#dashboard-screens").carousel();
+function hideNoticeBarOnClick (supportsLocalStorage, $noticeBar) {
+  $('#notice-bar-close').on('click', function (e) {
+    $noticeBar.addClass('hide');
+    if (supportsLocalStorage) {
```

Comment:
We can use a cookie it will works always.

Reply:
On the other hand we need additional lib for reading/writting cookies.

Reply:
Besides, localStorage can be used by ~95% users - check is purely for legacy browsers support, which we rather don't expect to show on out site.

Reply:
What lib `document.cooke` is good enough for adding one cookie :D